README.md

```diff
@@ -1,59 +1,21 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
-
```

Comment:
Please customize you README file to your own project. Kindly remove the screenshot of the other image and add yours